pkg/resource/plugin/plugin.go

```diff
@@ -55,6 +55,9 @@ type plugin struct {
 // pluginRPCConnectionTimeout dictates how long we wait for the plugin's RPC to become available.
 var pluginRPCConnectionTimeout = time.Second * 10
 
+// pluginRPCMaxMessageSize raises the gRPC Max Message size from `4194304` to `8388608`
+var pluginRPCMaxMessageSize = 1024 * 1024 * 8
```

Comment:
Why 2x vs  say 100x?

Seems 2x won’t make a huge dent in how common it is to hit this. 

Also - since we are doing gRPC within a single machine - I assume we are generally less sensitive to issues related to this limit?

Reply:
I was trying to be kinda conservative :D 

The userdata limit is 16KB, anything higher than that and AWS will reject it anyway. If you want, I can of course make it `1024*1024*100` - I am happy to do that if you want

Reply:
Yeah, we might as well make this pretty large if there are no adverse effects.